lib/cli/config.js

```diff
@@ -82,6 +85,12 @@ exports.loadConfig = filepath => {
       config = parsers.json(filepath);
     }
   } catch (err) {
+    if (isModuleNotFoundError(err)) {
+      throw createModuleNotFoundError(
+        `${filepath} is attempting to import a module that does not exist: ${err}`,
+        filepath
+      );
+    }
```

Comment:
It's not clear to me how you are going to distinguish `module not found` errors caused by invalid path (see `parser`) and by invalid imports.